static/app/types/profiling.d.ts

```diff
@@ -185,28 +187,28 @@ declare namespace Profiling {
   };
 
   type FrameInfo = {
-    key: string | number;
```

Comment:
I went ahead and sorted these to make it easier to parse given that we are starting to have quite a few